src/dearpygui_commands.h

```diff
@@ -1859,6 +1973,21 @@ show_tool(PyObject* self, PyObject* args, PyObject* kwargs)
 	return GetPyNone();
 }
 
+static PyObject*
+set_decimal_point(PyObject* self, PyObject* args, PyObject* kwargs)
+{
+	const unsigned int* point;
```

Comment:
:exclamation::exclamation: **BUG:** This picks up multiple characters from the string, and can set the separator to something like `0xdddd002c` instead of `0x2c` (comma). The pointer must be of the type `const char*`, not `const unsigned int*`.

Here's a test program I used to check that `set_decimal_point` really works (and yes, it fails with the `unsigned int*`):

```python
import dearpygui.dearpygui as dpg

dpg.create_context()
dpg.create_viewport(title='Test', width=600, height=300)

if hasattr(dpg, "set_decimal_point"):
    dpg.set_decimal_point(",")
else:
    print("WARNING: set_decimal_point is not supported.")

with dpg.window() as wnd:
    t1 = dpg.add_text()
    dpg.add_input_float(callback=lambda s, v: dpg.set_value(t1, str(v)))
    t2 = dpg.add_text()
    dpg.add_input_text(scientific=True, callback=lambda s, v: dpg.set_value(t2, str(v)))
    t3 = dpg.add_text()
    dpg.add_input_text(decimal=True, callback=lambda s, v: dpg.set_value(t3, str(v)))
    t4 = dpg.add_text()
    dpg.add_input_text(hexadecimal=True, callback=lambda s, v: dpg.set_value(t4, str(v)))

dpg.setup_dearpygui()
dpg.set_primary_window(wnd, True)
print(dpg.get_dearpygui_version())
print(dpg.__file__)
dpg.show_viewport()
dpg.start_dearpygui()
dpg.destroy_context()
```